ui/src/pages/CreatePoll.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { useState } from 'react';
+import PollBody from '../components/PollBody/PollBody';
+import PageLayout from '../components/PageLayout/PageLayout';
+import './style.css';
+
+const CreatePoll = () => {
+    const pollBody = () => {return <PollBody/>;}
```

Comment:
Pretty sure you can do this instead:
```suggestion
    return (
        <PageLayout>
            <PollBody />
        </PageLayout
    )
```
Looks cleaner, right?

Reply:
Yeah! Looks much cleaner now, thanks!